pkg/cloudevents/transport/http/transport_test.go

```diff
@@ -136,7 +136,7 @@ func TestStableConnectionsToSingleHost(t *testing.T) {
 func TestMiddleware(t *testing.T) {
 	testCases := map[string]struct {
 		middleware []string
-		want       string
```

Comment:
hm that is a weird change.

Reply:
Very! The want member is not used, I've no idea why I replaced it with "_" instead of just removing it. Removed.